### PR TITLE
Update README #4477

### LINKED ISSUE #4477: Update README
### Issue Description

🔑 

The README.md file does not have listings for the scan and supply tools. A bit of sorting/organization might be nice as well for iOS/Android pairings. The issue(s) call to action isn't linked.

### Environment

fastlane 1.83.0

_Do you use bundler to execute fastlane (i.e. `bundle exec fastlane`)?_ no
_Do you use a Ruby environment manager (e.g. `chruby`, `rbenv`, `rvm`)?_ no

### PR DESCRIPTION
**Pull request by bgannin**
Fixes #4477 Update README
- add link for issue reporting
- add entries for scan, screengrab, and supply tools
- minor grammar and formatting tweaks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,41 +62,43 @@ If you experience slow launch times of fastlane, try running:
...
-- `fastlane new_action`: Create a new action (integration) for fastlane
+- `fastlane new_action`: Create a new action *(integration)* for fastlane
 
 ## [`fastlane`](https://fastlane.tools) Toolchain
 
-Additionally to the `fastlane` commands, you now have access to these `fastlane` tools:
+In addition to `fastlane`'s commands, you also have access to these `fastlane` tools:
 
 - [`deliver`](https://github.com/fastlane/fastlane/tree/master/deliver): Upload screenshots, metadata, and your app to the App Store
+- [`supply`](https://github.com/fastlane/fastlane/tree/master/supply): Upload your Android app and its metadata to the Google Play Store
```
Comment (hemal) [inline README.md]: "Upload screenshots, metadata and your app to Google Play. Select tracks for builds and promote builds to production"

File: README.md [hunk 2]
```diff
@@ -111,10 +108,12 @@ Additionally to the `fastlane` commands, you now have access to these `fastlane`
 - [`pilot`](https://github.com/fastlane/fastlane/tree/master/pilot): The best way to manage your TestFlight testers and builds from your terminal
 - [`boarding`](https://github.com/fastlane/boarding): The easiest way to invite your TestFlight beta testers
 - [`gym`](https://github.com/fastlane/fastlane/tree/master/gym): Building your iOS apps has never been easier
-- [`match`](https://github.com/fastlane/fastlane/tree/master/match): Easily sync your certificates and profiles across your team using git
+- [`match`](https://github.com/fastlane/fastlane/tree/master/match): Easily sync your certificates and profiles across your team using Git
+- [`scan`](https://github.com/fastlane/fastlane/tree/master/scan): The easiest way to run tests of your iOS and Mac app
```
Comment (hemal) [inline README.md]: The easiest way to run tests for your iOS and Mac applications

File: README.md [hunk 3]
```diff
@@ -67,41 +62,43 @@ If you experience slow launch times of fastlane, try running:
...
+- Follow the assistant's prompts
+- Further customise the Ruby based `Fastfile` with additional [actions](https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Actions.md)
 
 For more details, check out the [fastlane guide](https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Guide.md).
 
-## Available commands
+## Available Commands
 
-Usually you'll use fastlane by triggering individual lanes:
+Typically you'll use `fastlane` by triggering your individual lanes:
```
Comment (hemal) [inline README.md]: remove "your" after triggering and before individual

File: README.md [hunk 4]
```diff
@@ -67,41 +62,43 @@ If you experience slow launch times of fastlane, try running:
...
 ## Quick Start
 
-The setup assistant will create all the necessary files for you, using the existing app metadata from iTunes Connect or Google Play.
+`fastlane`'s setup assistant creates all the necessary files to get you started, using existing app metadata from iTunes Connect or Google Play.
 
 - `cd [your_project_folder]`
 - `fastlane init`
-- Follow the setup assistant, which will set up `fastlane` for you
-- Further customise the `Fastfile` with [actions](https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Actions.md).
+- Follow the assistant's prompts
```
Comment (hemal) [inline README.md]: Follow the setup assistant to configure the app `Fastfile`

File: README.md [hunk 5]
```diff
@@ -67,41 +62,43 @@ If you experience slow launch times of fastlane, try running:
...
+- Further customise the Ruby based `Fastfile` with additional [actions](https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Actions.md)
```
Comment (hemal) [inline README.md]: Further customize the `Fastfile` with additional [actions]
(https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Actions.md)

File: README.md [hunk 6]
```diff
@@ -10,9 +10,9 @@ fastlane
 [![Gem](https://img.shields.io/gem/v/fastlane.svg?style=flat)](http://rubygems.org/gems/fastlane)
 [![Build Status](https://img.shields.io/circleci/project/fastlane/fastlane/master.svg?style=flat)](https://circleci.com/gh/fastlane/fastlane)
 
-`fastlane` is a tool for iOS and Android developers to automate tedious tasks like generating screenshots, dealing with provisioning profiles and releasing your application.
+`fastlane` is a tool for iOS, Mac, and Android developers to automate tedious tasks like generating screenshots, dealing with provisioning profiles, and releasing your application.
 
-Use a lane to define your process:
+Use a lane to define your particular process:
```
Comment (hemal) [inline README.md]: remove particular.

File: README.md [hunk 7]
```diff
@@ -21,37 +21,32 @@ lane :beta do
...
 
-To deploy a new beta version of your app
-```
-fastlane beta
+Then to deploy a new 'beta' version of your app just run
 ```
-
-To get started with `fastlane`, follow the [guide](https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Guide.md).
-
+fastlane beta```. *There's no step 3!* 😁
```
Comment (hemal) [inline README.md]: Instead of "There's no step 3!" should we put the 🚀?

**Comment by hemal** [comment]
This is awesome @bgannin. Can't wait to see these ship!

File: README.md (hunk 1 above)
Comment (bgannin) [inline README.md]: I pulled the "tagline" from each of the tool's READMEs as it seemed to be the pattern.

Comment (hemal) [inline README.md]: You're right, it looks like this is inaccurate on that README. I believe it's generally referred to as "Google Play" so we need to fix the supply README as well. I'll take a note to make sure we fix this on the README as well for the supply tool, nice eye! Can we add this description for now and we can fix the supply readme separately?

Comment (hemal) [inline README.md]: Let's shorten this for now to: "Upload screenshots, metadata and your app to Google Play"

File: README.md (hunk 5 above)
Comment (bgannin) [inline README.md]: My intent here was to make clear that it's not only actions that compose a Fastfile.

**Comment by bgannin** [comment]
PR feedback incorporated and pushed.

**Comment by bgannin** [comment]
Not sure why CircleCI failed… there are no code changes.

File: README.md [hunk 8]
```diff
@@ -21,37 +21,32 @@ lane :beta do
   match
   testflight
   sh "./customScript.sh"
-
```
Comment (KrauseFx) [inline README.md]: Why did you remove that newline? It looks nicer

File: README.md [hunk 9]
```diff
@@ -21,37 +21,32 @@ lane :beta do
...
+fastlane beta```. :rocket:
 
               |  fastlane
 --------------------------|------------------------------------------------------------
-:sparkles: | Connect all iOS and Android build tools into one workflow (both `fastlane` tools and third party tools)
-:monorail: | Define different `deployment lanes` for App Store deployment, beta builds or testing
-:ship: | Deploy from any computer, including a CI-server
+:sparkles: | Connect iOS, Mac, and Android build tools into one workflow (both `fastlane` tools and third party tools)
+:monorail: | Define different `deployment lanes` for app store deployment, beta builds, or testing
```
Comment (KrauseFx) [inline README.md]: Should be `App Store`

File: README.md (hunk 8 above)
Comment (bgannin) [inline README.md]: To me it felt rogue because all the actions are grouped in the lane/sequence except for slack. The lane's content is so compact it doesn't really necessitate whitespace. My 50 line lanes, however…

File: README.md (hunk 9 above)
Comment (bgannin) [inline README.md]: Oh right, I remember this change now… is this line meant to be iOS specific? (that's what uppercase is for, compared to the general distribution concept)

Comment (bgannin) [inline README.md]: Technically some of us aren't deploying to any app stores ;)

Comment (KrauseFx) [inline README.md]: It is iOS specific. For Android it's called `Google Play`

File: README.md (hunk 8 above)
Comment (KrauseFx) [inline README.md]: ok

File: README.md [hunk 10]
```diff
@@ -21,37 +21,32 @@ lane :beta do
...
 
-To deploy a new beta version of your app
-```
-fastlane beta
+Then to deploy a new 'beta' version of your app just run
 ```
-
-To get started with `fastlane`, follow the [guide](https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Guide.md).
-
+fastlane beta```. :rocket:
```
Comment (KrauseFx) [inline README.md]: Something is wrong here, this breaks markdown for the whole file

**Comment by KrauseFx** [comment]
Thanks for update the PR, I added a note about markdown being broken: https://github.com/bgannin/fastlane/blob/readme-update/README.md

Let me know when you had the time to update the PR so that I can do another review 👍

File: README.md (hunk 9 above)
Comment (bgannin) [inline README.md]: What I meant was is the entire sentence/concept of a deployment lane… the current terminology is iOS specific and some of us deploy, beta, and test _a)_ on other platforms (thus via an 'app store' somewhere), _b) _ some don't even use an app store :)

File: README.md (hunk 10 above)
Comment (bgannin) [inline README.md]: Don't know what happened there, will fix up now.

**Comment by bgannin** [comment]
Fixed the erroneous markdown. Should be good to go now.

**Comment by KrauseFx** [comment]
Seems like the build is failing, could you rebase from master and see if that fixes the issue? Sorry for the troubles

**Comment by bgannin** [comment]
@KrauseFx Should be good now.

**Comment by KrauseFx** [comment]
@hemal is this ready to be merged from your side?

File: README.md (hunk 9 above)
Comment (KrauseFx) [inline README.md]: Still iOS developers will complain about the spelling of this, let's go with `App Store` please

**Comment by hemal** [comment]
@KrauseFx LGTM 🚀

**Comment by KrauseFx** [comment]
Let's change this one last thing https://github.com/fastlane/fastlane/pull/4493#discussion_r62556260 and then we're ready to merge 👍 . You don't need to squash as we can do that on GitHub now 🚀

**Comment by bgannin** [comment]
@KrauseFx Done 💯

**Comment by KrauseFx** [comment]
Thanks, merged with https://github.com/fastlane/fastlane/pull/4600, I cherry-picked the commits 👍
